src/main/java/graphql/language/Directive.java

```diff
@@ -18,6 +18,8 @@
     private final String name;
     private final List<Argument> arguments = new ArrayList<>();
 
+    private static final String CHILD_ARGUMENTS = "arguments";
```

Comment:
not sure on the CHILD_ smurfing

The thing that represents "arguments" could be called arguments